test/unit_test/loss_functions_test.cc

```diff
@@ -40,3 +40,35 @@ BOOST_AUTO_TEST_CASE(squared_loss_test)
 
   VW::finish(vw);
 }
+
+BOOST_AUTO_TEST_CASE(expectile_loss_test)
```

Comment:
Please add more tests for
1. both cases of comparison between label and prediction
2. 0 and 1 for the loss parameter
3. comparison with squared loss

Reply:
I added these additional tests, now the expectile loss has 5 tests in total.